docs/source/en/_toctree.yml

```diff
@@ -83,6 +79,14 @@
     title: "How to add a model to 🤗 Transformers?"
   - local: add_new_pipeline
     title: "How to add a pipeline to 🤗 Transformers?"
+  - local: performance
+    title: Performance and scalability
+  - local: perf_train_gpu_one
+    title: Training on one GPU
+  - local: perf_train_gpu_many
+    title: Training on many GPUs
+  - local: perf_hardware
+    title: Custom hardware for training
```

Comment:
This ends up being a bit too low in the toc for my taste. I think at least the perforamce page should be where it was before.

Reply:
Moved it back up :)